internal/pkg/skuba/deployments/ssh/dex.go

```diff
@@ -58,3 +59,20 @@ func dexDeploy(t *Target, data interface{}) error {
 	_, _, err = t.ssh("kubectl --kubeconfig=/etc/kubernetes/admin.conf apply -f /tmp/dex.d")
 	return err
 }
+
+func dexRenewCertificate(t *Target, data interface{}) error {
+	_, _, err := t.ssh("sudo kubectl --kubeconfig=/etc/kubernetes/admin.conf delete secret oidc-dex-cert --namespace=kube-system")
+	if err != nil {
+		errors.Wrap(err, "unable to delete dex secret")
```

Comment:
This does not make any sense. Please only delete secret when it exists. The same as other place.